File: app/src/main/java/com/example/saferecorder/OpenActivity.java
package com.example.saferecorder;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class OpenActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_open);

        @SuppressLint("WrongViewCast") Button button = findViewById(R.id.lg_saferecorder_btn);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(OpenActivity.this, LoginActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/saferecorder/OpenActivity.kt
package com.example.saferecorder


import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.inappmessaging.model.Button


public class OpenActivity : AppCompatActivity() {

    private var btn_login: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_open)
        btn_login = findViewById<Button>(R.id.lg_saferecorder_btn)

        val private = null
        private Button btn_login;
        // 운전시작 버튼 클릭시 서비스 페이지로 이동
        btn_login.setOnClickListener(View.OnClickListener {
            val intent = Intent(this@OpenActivity, LoginActivity::class.java)
            startActivity(intent)
        })

    }
}

